uno.platform.uno-calculator.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: uno.platform.uno-calculator
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: start-uno-calculator.sh
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+modules:
+  - name: lzo
+    config-opts:
+      - "--enable-shared"
+      - "--disable-static"
+    cleanup:
+      - /include
+      - /share/doc
+      - "*.la"
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: "http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz"
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+        x-checker-data:
+          type: anitya
+          project-id: 1868
+          url-template: http://www.oberhumer.com/opensource/lzo/download/lzo-$version.tar.gz
+
+  - name: squashfs-tools
+    buildsystem: simple
+    build-commands:
+      - XZ_SUPPORT=1 LZO_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1
+    sources:
+      - type: git
+        url: https://github.com/plougher/squashfs-tools.git
+        tag: 4.6.1
+        commit: d8cb82d9840330f9344ec37b992595b5d7b44184
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\\d.]+)$
```

Comment:
Use both of these from shared modules https://github.com/flathub/shared-modules

Reply:
That's done now